R/methods-mzRpwiz.R

```diff
@@ -192,10 +191,11 @@ setMethod("chromatogramHeader", "mzRpwiz",
               } else {
                   stopifnot(is.numeric(chrom))
                   n <- nChrom(object)
-                  chrom <- as.integer(chrom)
```

Comment:
to be on the (very) save side I would suggest to then call `as.integer(chrom)` below (i.e. `getChromatogramHeaderInfo(as.integer(chrom))`)

Reply:
could be that the Rcpp makes sure the correct type is passed, but I would rather play save...

Reply:
Done.